Frontend/ui-driver/src/Resource/Constants.purs

```diff
@@ -257,12 +257,22 @@ convertCoinsToCashPopupEndTime = "16:30:00"
 dayEndTime :: String
 dayEndTime = "23:59:59"
 
-serviceTierMapping :: String -> Maybe Boolean -> String
-serviceTierMapping tierName acRide = 
-  case acRide, tierName of
-    Just true, "AC Mini" -> "Mini"
-    Just false, "Non-AC Mini" -> "Mini"
-    _ , _ -> tierName
+serviceTierMapping :: Boolean -> String -> Maybe Boolean -> String
+serviceTierMapping hideAcNonAc tierName acRide = 
+  case hideAcNonAc, acRide, tierName of
+    _ ,Just true, "AC Mini" -> "Mini"
+    false,Just true,"Ventilator" ->  "Ventilator"
```

Comment:
`false, Just _, "Ventilator" -> "Ventilator"`
We can reduce the  case matchings of such cases